packages/requisitions/src/utils.ts

```diff
@@ -29,9 +33,35 @@ export const placeholderSupplierRequisition: SupplierRequisition = {
   },
 };
 
-export const isRequisitionEditable = (
-  requisition: SupplierRequisition
-): boolean => {
+export const placeholderCustomerRequisition: CustomerRequisition = {
+  id: '',
+  requisitionNumber: 0,
+  isDeleted: false,
+  lines: [],
+  color: '#cdcdcd',
+  otherParty: {
+    id: '',
+    name: '',
+    code: '',
+    isCustomer: false,
+    isSupplier: true,
+  },
+  otherPartyId: '',
+  otherPartyName: '',
+  status: SupplierRequisitionNodeStatus.Draft,
```

Comment:
had a note in #540 about what to do with the status - currently `SupplierRequisitionNodeStatus` for both types of req